docs/readthedocs/source/doc/Friesian/QuickStart/Train_wnd.md

```diff
@@ -0,0 +1,269 @@
+
+    indicator_input_layers = []
+    indicator_layers = []
+    for i in range(len(column_info.indicator_cols)):
+        indicator_input_layers.append(tf.keras.layers.Input(shape=[], dtype="int32"))
+        indicator_layers.append(tf.keras.backend.one_hot(indicator_input_layers[i],
+                                                         column_info.indicator_dims[i] + 1))
+
+    embed_input_layers = []
+    embed_layers = []
+    for i in range(len(column_info.embed_in_dims)):
+        embed_input_layers.append(tf.keras.layers.Input(shape=[], dtype="int32"))
+        embedding_layer = tf.keras.layers.Embedding(column_info.embed_in_dims[i] + 1,
+                                                    output_dim=column_info.embed_out_dims[i])
+        iembed = embedding_layer(embed_input_layers[i])
+        flat_embed = tf.keras.layers.Flatten()(iembed)
+        embed_layers.append(flat_embed)
+
+    continuous_input_layers = []
+    continuous_layers = []
+    for i in range(len(column_info.continuous_cols)):
+        continuous_input_layers.append(tf.keras.layers.Input(shape=[]))
+        continuous_layers.append(
+            tf.keras.layers.Reshape(target_shape=(1,))(continuous_input_layers[i]))
+
+    if len(wide_base_layers + wide_cross_layers) > 1:
+        wide_input = tf.keras.layers.concatenate(wide_base_layers + wide_cross_layers, axis=1)
+    else:
+        wide_input = (wide_base_layers + wide_cross_layers)[0]
+    wide_out = tf.keras.layers.Dense(1)(wide_input)
+    if len(indicator_layers + embed_layers + continuous_layers) > 1:
+        deep_concat = tf.keras.layers.concatenate(indicator_layers +
+                                                  embed_layers +
+                                                  continuous_layers, axis=1)
+    else:
+        deep_concat = (indicator_layers + embed_layers + continuous_layers)[0]
+    linear = deep_concat
+    for ilayer in range(0, len(hidden_units)):
+        linear_mid = tf.keras.layers.Dense(hidden_units[ilayer])(linear)
+        bn = tf.keras.layers.BatchNormalization()(linear_mid)
+        relu = tf.keras.layers.ReLU()(bn)
+        dropout = tf.keras.layers.Dropout(0.1)(relu)
+        linear = dropout
+    deep_out = tf.keras.layers.Dense(1)(linear)
+    added = tf.keras.layers.add([wide_out, deep_out])
+    out = tf.keras.layers.Activation("sigmoid")(added)
+    model = tf.keras.models.Model(wide_base_input_layers +
+                                  wide_cross_input_layers +
+                                  indicator_input_layers +
+                                  embed_input_layers +
+                                  continuous_input_layers,
+                                  out)
+
+    return model
+```
+### **2.4. Distributed Training using Orca Estimator**
+Build an [Orca Estimator](../../Orca/Overview/distributed-training-inference.md), and `train_tbl.df` is then feed into the estimator to train the model.
```

Comment:
no need to show the full details; instead, just highlight that we can use `train_tbl.df` as input to the Orca Estimator